code/game/machinery/computer/crew.dm

```diff
@@ -133,6 +133,16 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 	var/life_status
 
 	for(var/mob/living/carbon/human/H in GLOB.carbon_list)
+		var/jammed = FALSE
+		var/turf/position = get_turf(H)
+		for(var/obj/item/jammer/jammer in GLOB.active_jammers)
+			var/turf/jammer_turf = get_turf(jammer)
+			if(position.z == jammer_turf.z && (get_dist(position, jammer_turf) <= jammer.range))
+				jammed = TRUE
+				break
+		if(jammed) // radio jammers prevent suit sensors 
```

Comment:
on a side note in the future you could make this a TRAIT_JAMMED instead of whatever this is